lib/uplink/uplink.go

```diff
@@ -82,6 +91,12 @@ func (cfg *Config) setDefaults(ctx context.Context) error {
 	if cfg.Volatile.Log == nil {
 		cfg.Volatile.Log = zap.NewNop()
 	}
+	if cfg.Volatile.DialTimeout.Seconds() == 0 {
+		cfg.Volatile.DialTimeout = 20 * time.Second
```

Comment:
maybe we can make `transport.defaultDialTimeout` public and use here? also for `RequestTime`. what do you think?

Reply:
Good idea! Done.